src/public/modules/forms/base/components/field-date.client.component.js

```diff
@@ -55,6 +56,9 @@ function dateFieldComponentController() {
       if (get(vm.field, 'dateValidation.customMaxDate')) {
         vm.dateOptions.maxDate = new Date(vm.field.dateValidation.customMaxDate)
       }
+
+      // Set default view upon opening datepicker to be month of minDate
+      vm.dateOptions.initDate = vm.dateOptions.minDate
```

Comment:
would it make more sense to execute this only when `vm.dateOptions.minDate` is set? i.e. in the if-block on L44

Reply:
I think it might have to be outside because `vm.dateOptions.minDate` is set in multiple places (L44, L53). Either that or set `initDate` after both L44 or L53.

Reply:
Yup Antariksh is right that it has to be after L44 and L53 - for clarity I wrapped in an `if` block so that initDate is set only if minDate